.github/workflows/pull_request.yml

```diff
@@ -26,6 +26,9 @@ jobs:
   # for each PR run integration test
   integration_test:
     uses: ./.github/workflows/integration_test.yml
+  # for each PR run ACPI validation
+  acpi_validation:
+    uses: ./.github/workflows/acpi-validation.yml
```

Comment:
as it's a PR level job, how do we ensure it build latest container image per PR? if the compose file just latest, so... the PR will use latest kepler image(which build in last night) and nothing checked.

Reply:
We are building Kepler as part of our compose: https://github.com/sustainable-computing-io/kepler/blob/878599d33cd0cd2eab49a8936890cf9e4ba06c54/manifests/compose/mock-acpi/compose.yaml#L11